src/Target.h

```diff
@@ -84,6 +84,7 @@ struct Target {
     enum Feature {
         JIT = halide_target_feature_jit,
         Debug = halide_target_feature_debug,
+        EnableBacktraces = halide_target_feature_enable_backtraces,
```

Comment:
Ahhh @mcourteaux - I'm so sorry to block you so late, but can you please add this to PyEnums.cpp, too, and leave a comment in both source locations to cross-check them?

Reply:
I'll change to an approve when you ACK this